docs/api/browser-window.md

```diff
@@ -222,6 +222,12 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       the top left.
     * `hiddenInset` - Results in a hidden title bar with an alternative look
       where the traffic light buttons are slightly more inset from the window edge.
+    * `overlay` - Results in a hidden title bar and a full size content window, yet
+       the title bar still has the standard window controls ("traffic lights" on macOS).
+       If `enableBlinkFeatures: 'WebAppWindowControlsOverlay'` is set,
```

Comment:
We should note here that this will become enabled by default in MXX / Electron XX?

Reply:
@MarshallOfSound there isn't a specific version of Chromium that this is currently targeted for.
https://fugu-tracker.web.app/ says that it is in Dev Trial in Chromium 92; Origin Trial in Chromium 93-95 but does not list an expected ship version.